### PR TITLE
Enable windows build and add ihnorton as maintainer

### PR DESCRIPTION
**Pull request by ihnorton**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ihnorton** [comment]
Also adds myself as a maintainer.

**Comment by jakebolewski** [comment]
👍

**Comment by jakebolewski** [comment]
I think you forgot to add the`build.bat` file?

**Comment by ihnorton** [comment]
lol, yes that would be necessary 🤦‍♂

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,8 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
- number: 0
- skip: True  # [win]
+ number: 1
```
Comment (jakebolewski) [inline recipe/meta.yaml]: I think you to re-render to skip msvc2008

ex from tiledb

```
  skip: True  # [win32]
  skip: True  # [win and vc<14]
  features:  # [win]
    - vc14  # [win]
```

**Comment by jakebolewski** [comment]
@ihnorton you need to re-render the recipe after you change the meta.yaml file

**Pull request merged**